lib/led/ledcontrol.js

```diff
@@ -674,7 +674,7 @@ const Controllers = {
           throw new Error("The `row` method is only supported for Matrix devices");
         }
         if (typeof val === "number") {
-          val = (`0000000000000000${parseInt(val, 10).toString(2)}`).substr(0 - (this.columns), this.columns);
+          val = (`0000000000000000${parseInt(val, 10).toString(2)}`).slice(-this.columns);
```

Comment:
I seem to recall that @rwaldron prefers the leading 0 before a negative variable for legibility, so I think we should bring it back here. I see this PR has been around for a while, so if you don't want to change it, then I can. 

Rick, please correct me if I remember it wrong.